docs/server/commands.rst

```diff
@@ -401,17 +402,46 @@ update_tmserver
 
 .. versionadded:: 2.7
 
+.. versionchanged:: 2.7.3 Renamed :option:`--overwrite` to :option:`--refresh`
+   and clarified the behavior of the command.
+
 Updates the ``default`` server in :setting:`POOTLE_TM_SERVER`.  The command
 reads translations from the current Pootle install and builds the TM resources
 in the TM server.
 
-By default the command will only add new translations to the server.  To
-rebuild the server from scratch use :option:`--rebuild`, this will completely
-remove the TM and rebuild it.  To ensure that the TM server remains available
-when you rebuild you can add :option:`--overwrite`.
+If no options are provided, the command will only add new translations to the
+server:
+
+.. code-block:: bash
+
+    $ pootle update_tmserver
```

Comment:
This command snippet adds no value. The same applies to the rest of the snippets.